### ISSUE TITLE
Portable build request

### ISSUE THREAD

**Issue by beesbeesbees2** [milkytracker/MilkyTracker#117]
Recently updated and found that Milkytracker now stores the config file in AppData instead of the program's directory. Would it be possible to either have this handled automatically (check the program directory first for a cfg and load/save there if appropriate before resorting to AppData) or to have a separate portable build with the old behavior?

**Comment by Deltafire** [comment]
MilkyTracker will store the config file in the directory pointed to by the APPDATA environment variable; this can be changed in a batch file or similar before loading the main executable.

**Comment by beesbeesbees2** [comment]
Won't that affect other programs running at the same time, though?

**Comment by Deltafire** [comment]
No, it will only affect programs started from within the batch file.  Here's an example:

Filename: mt.cmd, save in same directory as MilkyTracker.exe
```
set APPDATA=.
start MilkyTracker.exe
```

Double-click *mt.cmd*, this will start MilkyTracker and save it's config file inside the MilkyTracker subdirectory where the batch file is located.

**Comment by beesbeesbees2** [comment]
Good to know, thanks. Had to edit it a bit to work properly with drag and drop/'open with':
`set APPDATA=%~dp0
start %~dp0/MilkyTracker.exe %1`

It's still not quite ideal, but good enough for now.
